### ISSUE TITLE
`list` doesn't output app names

### ISSUE THREAD

**Issue by SHxKM** [mas-cli/mas#237]
**Describe the bug**
in the docs it says that `mas list` will return applications and their identifiers.

**To Reproduce**
1. `mas list`

**Expected behavior**
App name - App ID

**Actual behavior**
App ID (App ver)

Sample Output:

```
1225570693  (15.2)
1081413713  (3.9.5)
986304488  (2.0.17)
431511738  (1.8.1)
422304217  (1.10.6)
```

**Desktop (please complete the following information):**
 - mas version e.g. `10.14.4`
 - How did you install mas? `brew`

**Comment by phatblat** [comment]
This looks like a dup of #266 which was fixed in [1.6.3](https://github.com/mas-cli/mas/releases/tag/v1.6.3). What version of mas is this from?

**Comment by SHxKM** [comment]
@phatblat Thanks for the quick assistance.

Apparently brew will auto install `brew bundle` v `1.6.2` if Xcode isn't installed. Only after explicitly attempting to install `1.6.3` (using `--HEAD`) did it warn me that Xcode is needed for this version.
